bin/check-solr-replication-lag.rb

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env ruby
+#
+# Check how behind replication is.
+# ===
+#
+# Released under the same terms as Sensu (the MIT license); see LICENSE
+# for details.
+
+require 'sensu-plugin/check/cli'
+require 'rest-client'
+require 'json'
+require 'date'
+
+class SolrCheckReplication < Sensu::Plugin::Check::CLI
+  option :host,
+         short: '-h HOST',
+         long: '--host HOST',
+         description: 'Solr Host to connect to',
+         required: true
+
+  option :port,
+         short: '-p PORT',
+         long: '--port PORT',
+         description: 'Solr Port to connect to',
+         proc: proc(&:to_i),
+         required: true
+
+  option :core,
+         description: 'Solr Core to check',
+         short: '-d CORE',
+         long: '--core CORE',
+         required: true
+
+  option :warning,
+         description: 'Warning if greater than X seconds',
+         short: '-w SECONDS',
+         proc: proc(&:to_i),
+         default: 1200
+
+  option :critical,
+         description: 'Critical if greater than X seconds',
+         short: '-c SECONDS',
+         proc: proc(&:to_i),
+         default: 3600
+
+  option :unknown_core,
```

Comment:
this name implies to me that it will raise an unknown which is not the intention, I think something such as `core_missing_ok` would be clearer wdyt?

Reply:
changed.